pwiz_tools/Skyline/Model/AuditLog/AuditLogStrings.resx

```diff
@@ -633,7 +633,7 @@
     <value>The document was modified outside of Skyline: {0}</value>
   </data>
   <data name="AuditLogList_Validate_Audit_log_is_corrupted__Audit_log_entry_time_stamps_and_indices_should_be_decreasing" xml:space="preserve">
-    <value>Audit log is corrupted. Audit log entry time stamps and indices should be decreasing</value>
+    <value>Audit log is corrupted. Audit log entry time stamps and indices should be decreasing. Entry number {0} timestamp {1:G} occurs before entry number {2} timestamp {3:G}.</value>
```

Comment:
It's weird to say that the audit log entries should have their time stamps decreasing, since users would think of the audit log as being something in forward chronological order.

Maybe this error messsage could be changed to just say "in order" instead of either "decreasing" or "increasing".